src/stories/Button/Button.js

```diff
@@ -5,27 +5,27 @@ import {
   ButtonText,
 } from "./Button.style";
 
-const Button = ({ type, variant, text, icon }) => {
-  if (variant === "contained") {
+const Button = ({ type, shape, text, icon }) => {
+  if (shape === "contained") {
```

Comment:
@PRABgit01 button have variant props not shape

Reply:
done the changes